modules/react/modal/lib/ModalCard.tsx

```diff
@@ -1,18 +1,31 @@
 import * as React from 'react';
 
-import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {
+  createSubcomponent,
+  ExtractProps,
+  StyledType,
+  styled,
+} from '@workday/canvas-kit-react/common';
+import {space} from '@workday/canvas-kit-react/tokens';
 import {Popup} from '@workday/canvas-kit-react/popup';
 
 import {useModalCard, useModalModel} from './hooks';
 
 export interface ModalCardProps extends ExtractProps<typeof Popup.Card, never> {}
 
+const ResponsiveModalCard = styled(Popup.Card)<ModalCardProps & StyledType>(({theme}) => ({
+  margin: space.xl,
+  padding: space.l,
+  [theme.canvas.breakpoints.down('s')]: {
+    margin: `0 0 ${space.xl}`,
+    padding: space.m,
+  },
+}));
```

Comment:
The two biggest things that change are padding and margin. The padding decreases on smaller screen sizes and since the modal should transform from the bottom, I update the margin to give the modal some space.